util/mkinstallvars.pl

```diff
@@ -10,18 +10,24 @@
 # form, or passed as variable assignments on the command line.
 # The result is a Perl module creating the package OpenSSL::safe::installdata.
 
+use 5.10.0;
+use strict;
+use warnings;
+use Carp;
+
 use File::Spec;
-use List::Util qw(pairs);
+#use List::Util qw(pairs);
```

Comment:
shouldn't commented out code be removed?

Reply:
Nah, I leave it there to mark what I'm replacing with internal functionality